gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionSha256Sum=89d4e70e4e84e2d2dfbb63e4daa53e21b25017cc70c37e4eea31ee51fb15098a
```

Comment:
Currently, I use the following command to automatically generate the Gradle wrapper files for a version:

``` sh
gradle wrapper --gradle-version 8.11.1 --distribution-type all
```

Is there a command option to generate the checksum as well?

Reply:
Related: https://commonsware.com/blog/2023/01/28/applying-distributionSha256Sum-gradle-android-studio.html

Seems like, there is no command option. I checked: `gradle help --task wrapper` with Gradle 8.12.

Reply:
```
gradle wrapper --gradle-version 8.11.1 --distribution-type all \
  --gradle-distribution-sha256-sum 89d4e70e4e84e2d2dfbb63e4daa53e21b25017cc70c37e4eea31ee51fb15098a
```

Reply:
You can get the checksum from https://gradle.org/release-checksums/

Reply:
I see. I misunderstood `--gradle-distribution-sha256-sum`. Thank you.

Reply:
:heavy_check_mark: The checksum has been added in 977753829a5e3cea560d783c7d9d9d10ac43ef5f.